test/submission-access.js

```diff
@@ -6934,7 +6934,7 @@ module.exports = function(app, template, hook) {
         });
 
         it('An Admin, the owner, can update a submission owner, without explicit resource access (read)', function(done) {
-          tempSubmission.owner = null;
+          tempSubmission.owner = '';
```

Comment:
Had to change this because mongoose does not trigger a change when using virtual getter/setters when a null value is sent. Besides, from JSON it should appear as an empty string.